package.json

```diff
@@ -37,7 +40,7 @@
     "dirty-chai": "^2.0.1",
     "eslint": "^7.6.0",
     "eslint-config-digitalbazaar": "^2.5.0",
-    "eslint-plugin-jsdoc": "^30.2.0",
+    "eslint-plugin-jsdoc": "^32.3.0",
     "karma": "^5.1.1",
     "karma-babel-preprocessor": "^8.0.1",
```

Comment:
This and the other `@babel/*` packages above are not used (due to removal of karma babel processing step).